strings.xml

```diff
@@ -0,0 +1,82 @@
+<resources>
```

Comment:
Let's try removing these from the PR. In this case, you'll have to remove them locally and then also remove / delete them from git. Take a look here: https://stackoverflow.com/questions/2047465/how-can-i-delete-a-file-from-git-repo

Reply:
see comment on MainActivity.java